### PR TITLE
Fix Resources view issues

### PR DESCRIPTION
**Pull request by justin0022**
* Resolves https://github.com/tl-its-umich-edu/my-learning-analytics/issues/602 by setting 7 resources to be shown by default. 
* Resolves https://github.com/tl-its-umich-edu/my-learning-analytics/issues/592 by making sure domain of selection never hits `[0, 0]`.
* May have inadvertently resolved https://github.com/tl-its-umich-edu/my-learning-analytics/issues/577.

Also refactored/reorganized code.

### CODE REVIEW

File: assets/src/components/d3/createResourceAccessChart.js [hunk 1]
```diff
@@ -169,7 +169,11 @@ function createResourceAccessChart ({ data, width, height, domElement }) {
 
   function brushmove () {
     const fullRange = mainYZoom.range()
-    const selection = d3.event ? d3.event.selection : defaultSelection
+    const selection = d3.event
+      ? d3.event.selection[1] === 0 // prevents [0, 0] from being returned, which causes bug
```
Comment (justin0022) [inline assets/src/components/d3/createResourceAccessChart.js]: The bug was caused when the selection window's domain got to `[0, 0]`. I just check to make sure that the second element of the array is a 0, and if so set it to be small (0.1).

**Comment by pushyamig** [comment]
The UI looks good! Somewhere in the refractor, I see that resources URL link is broken. As part of the flow a user can click the link

**Comment by justin0022** [comment]
> The UI looks good! Somewhere in the refractor, I see that resources URL link is broken. As part of the flow a user can click the link

Ah good catch, I'll look into this.

**Comment by justin0022** [comment]
Links are fixed now @pushyamig

**Comment by pushyamig** [comment]
Screen Shot 2019-08-08 at 3 04 50 PM

The x-axis is showing the wrong %percent in decimal instead of integer.

**Comment by pushyamig** [comment]
Ok, Looks Good to me.

**Comment by jennlove-um** [comment]
Files view looks good.  Default to 7 bars works.  Bars are larger when there are fewer than 7 files, but that's okay.  Links to files work.

**Pull request merged**